user_docs/en/userGuide.t2t

```diff
@@ -2091,18 +2094,33 @@ Please see the display's documentation for descriptions of where these keys can
 | downArrow key | space+dot4 |
 | leftArrow key | space+dot3 |
 | rightArrow key | space+dot6 |
-| NVDA Menu | c1+c3+c4+c5 (command n) |
 | shift+tab key | space+dot1+dot3 |
 | tab key | space+dot4+dot6 |
 | alt key | space+dot1+dot3+dot4 (space+m) |
 | escape key | space+dot1+dot5 (space+e) |
 | enter key | dot8 |
-| windows+d key (minimize all applications) | c1+c4+c5 (command d) |
 | windows key | space+dot3+dot4 |
 | alt+tab key | space+dot2+dot3+dot4+dot5 (space+t) |
+%kc:endInclude
+
++++ Key assignments for Brailliant BI 32, BI 40 and B 80 +++ 

```

Comment:
Just to make sure, doesn't the BI 14 have these C keys?

Reply:
According to the documentation I have, BI 14 does not have command keys.